src/panels/lovelace/cards/hui-gauge-card.ts

```diff
@@ -230,14 +238,27 @@ class HuiGaugeCard extends LitElement implements LovelaceCard {
         max-width: 250px;
       }
 
+      .row {
+        display: flex;
+        justify-content: space-around;
+        padding-top: 4px;
+      }
+
       .name {
         text-align: center;
         line-height: initial;
-        color: var(--primary-text-color);
+        color: var(--secondary-text-color);
         width: 100%;
-        font-size: 15px;
+        font-size: 16px;
+        font-weight: 500;
         margin-top: 8px;
       }
+
+      .icon {
+        color: var(--state-icon-color, #44739e);
+        margin-top: 4px;
```

Comment:
Is this to center it vertically? You should set `align-items: center` on the row instead